### PR TITLE
General dependencies upgrade

### PR DESCRIPTION
**Pull request by symphony-thibault**
Upgraded outdated dependencies to their latest minor versions

### CODE REVIEW

**Review by symphony-elias — APPROVED** [review]

File: symphony-bdk-bom/build.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@ repositories {
 
 dependencies {
     // import Spring Boot's BOM
-    api platform('org.springframework.boot:spring-boot-dependencies:2.5.3')
```
Comment (symphony-youri) [inline symphony-bdk-bom/build.gradle:19]: it is also part of the spring examples

File: symphony-bdk-bom/build.gradle [hunk 2]
```diff
@@ -32,55 +32,55 @@ dependencies {
...
         api 'com.brsanthu:migbase64:2.2'
         api 'io.jsonwebtoken:jjwt:0.9.1'
-        api 'org.bouncycastle:bcpkix-jdk15on:1.67'
+        api 'org.bouncycastle:bcpkix-jdk15on:1.69'
         api 'com.google.code.findbugs:jsr305:3.0.2'
 
-        api 'io.github.resilience4j:resilience4j-retry:1.6.1'
+        api 'io.github.resilience4j:resilience4j-retry:1.7.1'
 
-        api 'com.fasterxml.jackson.core:jackson-databind:2.11.4'
```
Comment (symphony-youri) [inline symphony-bdk-bom/build.gradle:55]: this is a BOM for Jackson that we could use maybe

Comment (symphony-thibault) [reply symphony-bdk-bom/build.gradle:55]: Yep, done

File: symphony-bdk-bom/build.gradle (hunk 1 above)
Comment (symphony-thibault) [reply symphony-bdk-bom/build.gradle:19]: Also updated

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**